packages/utils/tokens/src/text-tokens.ts

```diff
@@ -19,19 +19,21 @@ export interface FontStyleTokens {
 export type FontTokens = FontStyleTokens & FontVariantTokens;
 
 export const fontStyles: TokenBuilder<FontTokens> = {
-  from: ({ fontFamily, fontSize, fontWeight, variant }: FontTokens, { typography }: Theme) => {
+  from: ({ fontFamily, fontLetterSpacing, fontLineHeight, fontSize, fontWeight, variant }: FontTokens, { typography }: Theme) => {
     const { families, sizes, weights, variants } = typography;
-    if (fontFamily || fontSize || fontWeight || variant) {
+    if (fontFamily || fontLetterSpacing !== undefined || fontLineHeight !== undefined || fontSize || fontWeight || variant) {
```

Comment:
Seems like all five of these should be consistent? Or is there something about the types of fontLetterSpacing/fontLetterHeight that require "!== undefined". Same thing below, why use ?? rather than ||

Reply:
Is it just because fontLetterSpacing/fontLetterHeight are optional

Reply:
I went with `!== undefined` because otherwise we would ignore defined-but-falsy values such as `{fontLetterSpacing: 0}`.

Honestly, we probably should be testing for being defined in these cases, as well as using `??` instead of `||` for this same reason.

Reply:
Resolved with 9f835ea and b96a19f.